lepton/volume.go

```diff
@@ -127,9 +127,14 @@ func (v *Volume) GetAll() error {
 	if err != nil {
 		return err
 	}
-	fmt.Println("NAME\t\t\tUUID\t\t\tPATH")
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"NAME", "UUID", "PATH"})
 	for _, vol := range vols {
-		fmt.Printf("%s\t\t\t%s\t\t\t%s\n", vol.Name, vol.ID, vol.Path)
+		var row []string
+		row = append(row, vol.Name)
+		row = append(row, vol.ID)
+		row = append(row, vol.Path)
+		table.Append(row)
 	}
 	return nil
```

Comment:
did you try this out? looks like it's missing a table.Render...

Reply:
@eyberg 
Sorry, it was missing to git add. But I added it right now.